library/psa_crypto_driver_wrappers.c

```diff
@@ -1995,4 +1995,14 @@ psa_status_t psa_driver_wrapper_free_random(
     return PSA_SUCCESS;
 }
 
+psa_status_t psa_driver_wrapper_destroy_key(const psa_key_attributes_t *attributes)
+{
+        psa_key_location_t location = PSA_KEY_LIFETIME_GET_LOCATION(attributes->core.lifetime);
+
+        switch (location) {
+        }
```

Comment:
?

Reply:
implementation is in nrf.

Reply:
Can we give context to the empty switch by adding a comment?

I don't have any good suggestions off the top of my head

Something along the lines of:
Add code logic to destroy key if location matches with a given driver